sympy/integrals/integrals.py

```diff
@@ -519,12 +520,14 @@ def try_meijerg(function, xab):
 
                         function = antideriv._eval_interval(x, a, b)
                         function = Poly(function, *gens)
-                    elif isinstance(antideriv, Add):
-                        function = Add(*[i._eval_interval(x,a,b) for i in
-                            Add.make_args(antideriv)])
                     else:
                         try:
-                            function = antideriv._eval_interval(x, a, b)
+                            if b is S.Infinity:
```

Comment:
This test and the following lines are not necessary as the limit is taken into account in ``_eval_interval``. They seem to change the result in a way that makes some tests fail.